script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -87,27 +87,27 @@ Of course, I'm not praising her in the slightest.
 [sha:aba59f1918a4eec4b4cdfc6a3c06287c6bedfdb7]{
 -- Page 429, Offset 9719.
 -- 「ああでも凄いなクソ、偶然か、それとも計算か!?」
-"But damn, that was cool. Was that by accident, or was it part of her plan!?"
+"But damn, that was cool. Was that an accident, or was it part of her plan!?"
 }
 [sha:42f89456878a9acd46e8c730a1135de7c2f6c49c]{
 -- Page 429, Offset 9720.
 -- 屋上から見るかぎり死傷者はでていない。
-There aren't any casualties as best I can tell from the rooftop.
+As far as I can tell from the rooftop, there aren't any casualties.
```

Comment:
nit: "casualties" refers to both dead and injured. "fatalities" would be my suggestion